lib/Drush/Boot/BaseBoot.php

```diff
@@ -62,6 +62,10 @@ function bootstrap_and_dispatch() {
           $command += $this->command_defaults();
           // Insure that we have bootstrapped to a high enough
           // phase for the command prior to enforcing requirements.
+          if ($command['command'] == 'updatedb-batch-process') {
```

Comment:
You've beat me here .. I am not familiar with the drush architecture that well and those were my first baby steps .. So I guess probably someone else could take a deeper look into it?